app/messages/summaries.email.html.erb

```diff
@@ -11,17 +11,19 @@
 <% end %>
 
 <% define_content :footer_link do %>
+<!--
```

Comment:
On Thu, Sep 17, 2015 at 09:39:41AM -0700, Brian Sadler wrote:
> Maybe I'm taking crazy pills, but it looks like you commented out the specific communication profile one and left the generic footer_link.

That defines a little inline function that is called later in the layout. It resolves to a link to the communication profile, same as the one next to in the final thing but separate in the source.

It'll probably make a whole lot more sense once we send one of these emails...